hare/hare.go

```diff
@@ -47,10 +47,14 @@ type Consensus interface {
 type TerminationOutput interface {
 	ID() types.LayerID
 	Set() *Set
-	Coinflip() bool
 	Completed() bool
 }
 
+type WeakCoinOutput interface {
+	ID() types.LayerID
+	Coinflip() bool
+}
```

Comment:
can't it be just a struct? whats the point of interface?

Reply:
yeah. was thinking of minimizing the change while keeping parallel to termination output (and reusing the same impl for test).
now changed both outputs to struct.